### DISCUSSION TITLE
Hey, you just forgot to copy my latest EFI

### DISCUSSION THREAD

**Discussion by RobyRew** [wilsomwong/Asus-TUF-FX504GE-Hackintosh#8] [Ideas]
I sow that you've been copying my EFIs overtime, I understand "you want to have your own copy of the EFI I built or WHATEVER is your plan", but at the very least put the credits or indicate that all your EFIs come from my repository:
https://github.com/RobyRew/ASUS-FX504GE-Hackintosh_OpenCore

I'm not trying to be rude, but i'm saying this for the people that might encounter this repo and find no help, I hope you understand and do the right thing.

Cheers!

**Comment by wilsomwong** [comment]
I think the act of stealing and calling myself a creator is an abominable behavior, and I don't encourage these things to happen, but this time my behavior really displeased you, so I'm deeply sorry and promise to add credits to the macOS Ventura file. If you are still dissatisfied, you can tell me your dissatisfaction or ask me to delete it, and I will follow your choice. Of course, if you don't ask me to delete it, I will solve the problems in your EFI one after another and I am willing to work with you to create the most perfect EFI. I believe that what the Hackintosh community needs are a society that can help each other to solve problems in order for this society to continue. This is the meaning of the existence of Hackintosh. Thank you for reminding me that I made a mistake and I promise not to do it again next time.

**Reply by RobyRew** [reply]
yeah, if you want to help I welcome everyone, if you would aport help, you would be Credited, be sure about that.

Otherwise if you want to maintain this repo at least say somewhere on the readme that my EFIs builds helped you or they are used as a base for yours.

but in the end i can't force you, my intention is to help people , heck i don't ask for donations or nothing similar on any repo, hope you understand.

cheers!

**Comment by wilsomwong** [comment]
Oh, I seem to have misunderstood what you're trying to convey, so I seem to be off-topic by writing so many words. In short, I will give the credits to more people for reference, thank you for reminding me of my mistakes, and being willing to give me the opportunity to correct them.
